src/app/routes/public/public-contact/public-contact.component.html

```diff
@@ -62,7 +62,7 @@ <h3 class="flex flex-middle margin-remove">
                   </h3>
                   <mat-accordion>
                     <mat-expansion-panel (opened)="panelOpenState = true" (closed)="panelOpenState = false"
-                      *ngFor="let helpContent of chelp.contents | filter : searchText">
```

Comment:
please check it should not get removed

Reply:
No Vishnu, with that code, It is not working as per the expectation.